spec/game_stats_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'spec_helper.rb'
+
+RSpec.describe LeagueStats do
+  before(:each) do
+    @game_path = './data/games_fixture.csv'
+    @team_path = './data/teams.csv'
+    @game_teams_path = './data/game_teams_fixture.csv'
+    @locations = {
+        games: @game_path,
+        teams: @team_path,
+        game_teams: @game_teams_path
+    }
+    @stat_tracker = StatTracker.from_csv(@locations)
+    end
+
+    describe "#count_of_games_by_season" do
+        it "can count total games in one season" do
+            expect(@stat_tracker.count_of_games_by_season).to eq ({"20122013" => 10})
+        end
+
+    end
+
+    describe "#average_goals_per_game" do
+        it "can determine average goals of a all games" do
+            expect(@stat_tracker.average_goals_per_game).to eq (1.85)
+        end
+    end
+
+    describe "#average_goals_by_season" do
+        it "can determine average goals of all games in a given season" do
+            expect(@stat_tracker.average_goals_per_game).to eq ({"20122013" => 1.85, "20162017" => 2.50 })
```

Comment:
line 31 should have @stat_tracker.average_goals_by_season instead of .average_goals_per_game. I did a similar thing when copying code.

Reply:
Thanks for pointing that out! Yeah reusing existing formats for specs can leave your file a little lacking.